src/index.ts

```diff
@@ -208,7 +208,7 @@ export class Upload extends Pumpify {
   private bufferStream?: PassThrough;
   private offsetStream?: PassThrough;
 
-  private get baseURI() {
+  private get baseURI(): string {
```

Comment:
I'm kinda shocked this was creating problems 🤔

Reply:
Me too actually, it looks like the declaration file just defined this as

```js
private get baseUri();
```